src/_cffi_src/openssl/x509name.py

```diff
@@ -39,7 +39,10 @@
 /* These became const X509_NAME * in 1.1.0 */
 int X509_NAME_entry_count(X509_NAME *);
 X509_NAME_ENTRY *X509_NAME_get_entry(X509_NAME *, int);
+/* manual for X509_NAME_PRINT_EX ``strongly discourages'' the use of
+   X509_NAME_oneline */
```

Comment:
Can you remove this comment, we don't generally include commentary on the APIs in the bindings.

Reply:
Done in 4eb414c